velox/docs/functions/spark/math.rst

```diff
@@ -143,7 +143,7 @@ Mathematical Functions
 .. spark:function:: floor(x) -> [same as x]
 
     Returns ``x`` rounded down to the nearest integer.
-    Supported types are: BIGINT and DOUBLE.
+    Supported types are: BIGINT,  DOUBLE and DECIMAL.
```

Comment:
Would you document the decimal floor function in https://github.com/facebookincubator/velox/blob/9e418b172c7b316d351acd6c4b3428d5ca03eebd/velox/docs/functions/spark/decimal.rst  perhaps as below?

https://github.com/facebookincubator/velox/blob/9e418b172c7b316d351acd6c4b3428d5ca03eebd/velox/docs/functions/presto/decimal.rst?plain=1#L252